content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -483,10 +517,17 @@ ecosystem, such as [OPA Gatekeeper](https://github.com/open-profile-agent/gateke
 ### What profiles should I apply to my Windows Pods?
```

Comment:
Let's mention that HostProcess is alpha here, in case readers infer that it's a stable feature.